common/ruletypes.h

```diff
@@ -1009,6 +1125,30 @@ RULE_CATEGORY_END()
 RULE_CATEGORY(Command)
 RULE_BOOL(Command, DyeCommandRequiresDyes, false, "Enable this to require a Prismatic Dye (32557) each time someone uses #dye.")
 RULE_BOOL(Command, HideMeCommandDisablesTells, true, "Disable this to allow tells to be received when using #hideme.")
+RULE_INT(Command, MaxHelpLineLength, 53, "Maximum length of a line before splitting it in to new lines for DiaWind. Default 53.")
+RULE_STRING(Command, DescriptionColor, "light_grey", "Color for command help windows")
+RULE_STRING(Command, DescriptionHeaderColor, "indian_red", "Color for command help windows")
+RULE_STRING(Command, AltDescriptionColor, "light_grey", "Color for command help windows")
+RULE_STRING(Command, NoteColor, "dark_orange", "Color for command help windows")
+RULE_STRING(Command, NoteHeaderColor, "indian_red", "Color for command help windows")
+RULE_STRING(Command, AltNoteColor, "dark_orange", "Color for command help windows")
+RULE_STRING(Command, ExampleColor, "goldenrod", "Color for command help windows")
+RULE_STRING(Command, ExampleHeaderColor, "indian_red", "Color for command help windows")
+RULE_STRING(Command, SubExampleColor, "slate_blue", "Color for command help windows")
+RULE_STRING(Command, AltExampleColor, "goldenrod", "Color for command help windows")
+RULE_STRING(Command, SubAltExampleColor, "goldenrod", "Color for command help windows")
+RULE_STRING(Command, OptionColor, "light_grey", "Color for command help windows")
+RULE_STRING(Command, OptionHeaderColor, "indian_red", "Color for command help windows")
+RULE_STRING(Command, SubOptionColor, "light_grey", "Color for command help windows")
+RULE_STRING(Command, AltOptionColor, "light_grey", "Color for command help windows")
+RULE_STRING(Command, SubAltOptionColor, "light_grey", "Color for command help windows")
+RULE_STRING(Command, ActionableColor, "light_grey", "Color for command help windows")
+RULE_STRING(Command, ActionableHeaderColor, "indian_red", "Color for command help windows")
+RULE_STRING(Command, AltActionableColor, "light_grey", "Color for command help windows")
+RULE_STRING(Command, HeaderColor, "indian_red", "Color for command help windows")
+RULE_STRING(Command, SecondaryHeaderColor, "slate_blue", "Color for command help windows")
+RULE_STRING(Command, AltHeaderColor, "indian_red", "Color for command help windows")
+RULE_STRING(Command, FillerLineColor, "dark_grey", "Color for command help windows")
```

Comment:
This feels like a bit much to have this level of customization for display command in. We should just have sane defaults for things like this